main.go

```diff
@@ -201,6 +201,11 @@ func getConfig(ctx context.Context) (*api.APIConfig, *api.APIConfig, crypt.Secre
 
 	configReader := api.NewConfigReader(secretHelper)
 
+	for !foundation.FileExists(*configFilePath) {
```

Comment:
Let's test these changes first, then we can improve this later by checking a bit more frequently so it can start faster if possible.